### PR TITLE
fix: resolve pathing bug and add test coverage

### PR DESCRIPTION
**Pull request by jesseproudman**
Fixes a bug around missing load paths in the prior release that wasn't covered in test coverage.

### CODE REVIEW

File: src/testTrack.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import TestTrack from './testTrack';
+import ConfigParser from './configParser';
```
Comment (zerodom30) [inline src/testTrack.test.js:2]: The lint check on this line needs to be disabled.

**Review by zerodom30 — CHANGES_REQUESTED** [review]

**Review by zerodom30 — APPROVED** [review]
platformLGTM

**Review by zerodom30 — APPROVED** [review]
domainLGTM

**Comment by jesseproudman** [comment]
I don't have merge rights so we'll need to do the build / release process again. :(

**Pull request merged**